### ISSUE TITLE
Can't disconnect PayPal account for Payouts

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#4483]
In `Edit` > `Sending Money`, you have an option to Enable/Disable PayPal but there's no way to disconnect it. It's a problem for support as we sometimes want people to re-connect their PayPal. They may also want to switch to another account.

![image](https://user-images.githubusercontent.com/1556356/126599350-eb41d23f-1aed-4441-a639-275e86e52535.png)

**Comment by kaylarep** [comment]
Please ping if this is unrelated, and I can make it its own issue:

## [User Story ](https://opencollective.freshdesk.com/a/tickets/81304)
> Our team has brought up is the ability to enable or disable the possibility of using Paypal. As of right now, our fiscal host __ has issues with Paypal integration.  They've told us they can no longer use that as a payment option. However, as the option is still on our page people will often choose to apply using the email option. This means we have to have a conversation with them regarding it and ask them to submit their bank details. If we could remove the Paypal option it would save a lot of time and mean people could be paid quickly.

**Comment by Betree** [comment]
> Please ping if this is unrelated, and I can make it its own issue:
> ## [User Story ](https://opencollective.freshdesk.com/a/tickets/81304)
> 
> > Our team has brought up is the ability to enable or disable the possibility of using Paypal. As of right now, our fiscal host __ has issues with Paypal integration.  They've told us they can no longer use that as a payment option. However, as the option is still on our page people will often choose to apply using the email option. This means we have to have a conversation with them regarding it and ask them to submit their bank details. If we could remove the Paypal option it would save a lot of time and mean people could be paid quickly.

@kaylarep It is already possible to disable PayPal today. In the linked ticket, the issue seems to be that the fiscal host has issues with PayPal that they're currently working on and haven't disabled while they do. I don't think there's a problem on the platform itself, the fiscal host should just have disabled PayPal if not supported.

This issue is about being able to disconnect your account in order to re-connect it later.

**Comment by SudharakaP** [comment]
I've tested this and I think we don't show the UI buttons correctly here when we create a new host. 

When a new organization is created and activated as a host and when we got to the Sending Money section what we see is,

![image](https://user-images.githubusercontent.com/12435965/193739162-81aeb2fb-81ba-49ff-80be-0a70019faaf5.png)

Note that its as if PayPal Payouts are enabled by default. I think this is incorrect since PayPal Payouts are only enabled when we set the feature flag `settings.features.paypalPayouts` to `true`. 

I suggest the following solution. 

1) When a new Organization is created and activated as a host we show a info message instead of this button under the `PayPal` section (under `Sending Money`). This will be something along `Please contact `support@opencollective.com` if you like to activate PayPal Payouts for sending money`. 

2) If the user has connected PayPal Payouts (after we have set the feature flag, and the user has filled up the client secret and id as per, https://docs.opencollective.com/help/fiscal-hosts/payouts/payouts-with-paypal), then we show two buttons; `Disable PayPal` and `Disconnect PayPal`. 

3) If the user click on `Disable PayPal` we just disable PayPal payouts by setting the `disablePaypalPayouts` to `true` and a new button `Enable PayPal` would appear. If the user click on `Disconnect PayPal` we show the PayPal client ID and secret fields so that the user can specify the information for a new PayPal account. 

cc: @Betree

**Comment by Betree** [comment]
@SudharakaP Yes. When "Disconnecting", we should also cancel all PayPal recurring contributions (that needs to be cristal clear in the UI). Also keep in mind https://github.com/opencollective/opencollective/issues/4210.

**Comment by SudharakaP** [comment]
> @SudharakaP Yes. When "Disconnecting", we should also cancel all PayPal recurring contributions (that needs to be cristal clear in the UI). Also keep in mind #4210.

@Betree : Thanks for pointing that out. Currently the [disconnect button we have](https://github.com/opencollective/opencollective-frontend/blob/0f0716315381e187d5b9721e1cef06845d290d0d/components/edit-collective/EditPayPalAccount.js#L85-L88) do not support cancelling all recurring payments. I've opened a PR to fix the issue mentioned here, and we'll iterate on disconnecting the recurring contributions on a separate issue to keep things simple if there's no objections. However I have one concert about this matter; say if the user just wants to update their client-id/secret without cancelling all recurring contributions? Maybe we should give that option to user to choose to cancel their recurring contributions or not (maybe a confirmation modal that has a checkbox asking the user if they want to cancel recurring contributions as well)?

**Comment by SudharakaP** [comment]
We have discussed that this will be addressed as part of https://github.com/opencollective/opencollective/issues/4210 as this is much more complicated and needs to handle adaptive payment enabling/disabling as well.

**Comment by BenJam** [comment]
related #4627
